prov/efa/src/rxr/rxr_ep.c

```diff
@@ -302,6 +302,37 @@ int rxr_ep_post_prov_buf(struct rxr_ep *ep, uint64_t flags, enum rxr_lower_ep_ty
 	return 0;
 }
 
```

Comment:
Typo in commit ```prov/efa: adjust internal buffer posting in zero copy receive mode```

If application posted recevie buffer -> If the application posted a receive buffer,

Reply:
fixed